### DISCUSSION TITLE
alist-aria2 docker volumes 映射aira2配置文件失败

### DISCUSSION THREAD

**Discussion by zaixianyonghu** [AlistGo/alist#4985] [Q&A]
alist-aria2 docker volumes 映射aira2配置文件失败，在这个容器里 将aria2的配置文件映射到宿主机上，docker启动后aira2的配置文件并没有映射到宿主机上，aira2启动失败
映射： - /opt/docker_data/aria2/config:/root/.aria2

**Comment by li-peifeng** [comment]
我记得aria2 不是这路径吧？ 好像是/config
